source/documentation/using_the_api/register_components.md

```diff
@@ -1,18 +1,34 @@
-## The relationship between entries, items and records
+## The components of a register
 
-Registers are made up of:
+Registers are primarily composed of:
 
 * entries
 * items
 * records
 
+### Entries
+
 Entries contain metadata about what and when data has changed. 
 
-Each entry in a register has a relationship with one item, which contains data. An item is connected to an entry with an item hash.   
+Entries contain the fields `entry-number`,  `entry-timestamp`, `item-hash` and `key`. 
+
+The `entry-number` is unique and defines an entry’s position within the ordered list of a register. The `entry-timestamp` is the time when an entry was introduced to a given register, but it is no guarantee of the order of entries in a register. The `entry-number` is.
+
+Entries are connected to items by a given `item-hash`. 
+
+Each `key` is an alphanumeric string unique to the data in a register. The primary key (i.e. the unique identifier) always has the same name as the name of the register. For example, every entry in the `local-authority-eng` register has a field called `local-authority-eng`. Foreign keys, which are not always present, link to other registers. 
+
+> Entries also contain the field `index-entry-number`, which is a unique number that defines an entry's position within the ordered list of an index. Indexes are not a reliable feature yet, which are not ready for use by most users.  
```

Comment:
What about putting the warning in another paragraph, on the lines of:

> **Note**: `index-entry-number` depends on indexes, an experimental feature. Use it at your own risk.

Reply:
I would like to keep the alpha/beta labels to a minimum, we've been burnt before by multiple labels for different things.

I don't know what is the GOV.UK style for warnings but if it expresses clearly that the feature is not reliable because it is experimental, I don't mind.

Reply:
No problem. How about keeping it relatively simple for now:

> `index-entry-number` depends on indexes, which are an experimental and unreliable feature. 

If I have to change this after a writer review, I will do.